apps/coscrad-frontend/src/components/resources/factories/resource-detail-factory.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { ResourceType } from '@coscrad/api-interfaces';
+import { ResourceDetailContainer } from './resource-detail-conainer';
+
+export interface IResourceDetailFactory<T> {
```

Comment:
What is the purpose of the generic? Can we flow it through to the `ResourceDetailContainer<T>`?

Reply:
done